python-sdk/docs/development/DEVELOPMENT.md

```diff
@@ -202,3 +202,82 @@ You can configure the Docker-based testing environment to test your DAG
 
 * `dev/dags/` - DAG Files
 * `dev/logs/` - Logs files of the Airflow containers
+
+
+## Adding support for a new database
+
+You can use Test Driven approach for adding support for a new database to the Astro Python SDK. You can fulfil the
+[tests](https://github.com/astronomer/astro-sdk/tree/main/python-sdk/tests) for all the
```

Comment:
For referencing internal files, use [relative links](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links).

Reply:
done, thanks.